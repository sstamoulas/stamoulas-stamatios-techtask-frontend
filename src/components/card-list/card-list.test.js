import React from 'react'
import { create } from 'react-test-renderer'
 
import CardList from './card-list.component'

describe('Card List Component', () => {
  let component
  let ingredients = ['Bread', 'Cheese', 'Butter']
 
  beforeEach(() => {
    component = create(<CardList ingredients={ingredients} />)
  })
 
  it('should render with given props', () => {
    expect(component.toJSON()).toMatchSnapshot()
  })
})
