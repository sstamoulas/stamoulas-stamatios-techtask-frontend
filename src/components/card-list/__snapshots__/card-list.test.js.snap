// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card List Component should render with given props 1`] = `
<div
  className="flip-card-back"
>
  <div>
    Ingredients:
  </div>
  <ul
    className="list-container"
  >
    <li
      className="card-list-item"
    >
      Bread
    </li>
    <li
      className="card-list-item"
    >
      Cheese
    </li>
    <li
      className="card-list-item"
    >
      Butter
    </li>
  </ul>
</div>
`;
